lib/modules/apostrophe-svg-sprites-widgets/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  extend: 'apostrophe-pieces-widgets',
+  label: 'Svg Widget',
+  piecesModuleName: 'apostrophe-svg-sprites'
```

Comment:
I wonder if this was needed? Looks like it should find it on its own, but it's OK to specify it.

Reply:
removed